frontend/src/components/HomePage.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import ListCar from "./ListeCar";
+import NavBar from "./NavBar";
+import Historic from "./Historic";
+
+function HomePage() {
+  return (
+    <div>
+      <ListCar />
+      <NavBar />
```

Comment:
Shouldn't the `<NavBar>` be present in every page ? If so, I think it should be called from the `<App>` component

Reply:
review done

Reply:
we don't really want to see the navbar on every page ex Login PAGE, chargement page, confirmationPage(new car, new km)...
so I think it should be in a child of <App>